frontend/gulp/common.js

```diff
@@ -8,6 +8,13 @@
 
 var gutil = require('gulp-util');
 
+exports.argv = require('yargs').argv;
```

Comment:
Consider using process.argv http://stackoverflow.com/questions/4351521/how-do-i-pass-command-line-arguments-to-node-js